### ISSUE TITLE
Set URI for the created model

### ISSUE THREAD

**Issue by Gilwe** [react-monaco-editor/react-monaco-editor#324]
**TL;DR**, is it possible to add uri to the props and use it in the `createModel()`s?

Hi,

So, im trying to connect a react version of monaco with a remote language server.
I'm using  https://github.com/TypeFox/monaco-languageclient in order to set it up in order to connect to the lang server.

Generally, the language client notifies the server about a new file with LSP (language server protocol)'s init message:

```
{
    method: "textDocument/didOpen"
    params: {,…}
    textDocument: {uri: "file:///users/gilad/monaco-languageclient/example/lib/python3/main.py", languageId: "python",…}
    languageId: "python"
    text: <The file's value>
    uri: "file:///users/myuser/monaco-languageclient/example/lib/python3/main.py"
}
```

It derives the language, value and uri from the monaco's model.
The problem is that react-monaco-editor doesn't init the uri parameter in: https://github.com/react-monaco-editor/react-monaco-editor/blob/85af6495c952c3d700fab7dbd0c0ecc579a3bb0a/src/diff.tsx#L129-L137

as it doesn't use the third argument:
`export function createModel(value: string, language?: string, uri?: Uri): ITextModel;`

So the simple solution is to have a uri in the props which will be used in the above lines, it will populate the uri in monaco's model and should have no other effect in this repo's behavior.

Thank you.
